CHANGELOG.md

```diff
@@ -4,10 +4,20 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## [Unknown]
+## 0.3.0
 
 ### Added
 - Function documentation comment completion. Type "///" on line above function to activate. 
+- Right click menu to package dependency view. Entries include
```

Comment:
```suggestion
- Right click menu to see package dependency view. Entries include:
```

Reply:
The right click menu isn't to see the dependency view. It is to add options for dependencies in it eg use local version, view repository

Reply:
Should this be for or add instead then? The sentence confused me

Reply:
I'll reword

"Package dependency view has new right click menu. Menu entries include."